mappings/net/minecraft/client/render/item/ItemRenderer.mapping

```diff
@@ -62,11 +62,18 @@ CLASS net/minecraft/class_918 net/minecraft/client/render/item/ItemRenderer
 		ARG 2 stack
 		ARG 3 x
 		ARG 4 y
+	METHOD method_27952 getArmorVertexConsumer (Lnet/minecraft/class_4597;Lnet/minecraft/class_1921;ZZ)Lnet/minecraft/class_4588;
+		ARG 3 glint
 	METHOD method_27953 renderInGui (Lnet/minecraft/class_1799;II)V
 		COMMENT Renders an item in a GUI without an attached entity.
 		ARG 1 stack
 		ARG 2 x
 		ARG 3 y
+	METHOD method_29711 getDirectGlintVertexConsumer (Lnet/minecraft/class_4597;Lnet/minecraft/class_1921;ZZ)Lnet/minecraft/class_4588;
+		ARG 1 layer
+		ARG 3 glint
+	METHOD method_30114 getTransformingGlintVertexConsumer (Lnet/minecraft/class_4597;Lnet/minecraft/class_1921;Lnet/minecraft/class_4587$class_4665;)Lnet/minecraft/class_4588;
+	METHOD method_30115 getTransformingDirectGlintVertexConsumer (Lnet/minecraft/class_4597;Lnet/minecraft/class_1921;Lnet/minecraft/class_4587$class_4665;)Lnet/minecraft/class_4588;
```

Comment:
If you could, it would be nice to describe how the `direct` one renders straight to the frame buffer.

Reply:
Imo that's extraneous.